frontend/src/features/admin-form/settings/SettingsEndPage.tsx

```diff
@@ -0,0 +1,19 @@
+import { Skeleton } from '@chakra-ui/react'
+
+import { useAdminForm } from '../common/queries'
+
+import { CategoryHeader } from './components/CategoryHeader'
+import { EndPageSettingsInput } from './components/EndPageSettingsSection/EndPageSettingsInput'
+
+export const SettingsEndPage = (): JSX.Element => {
+  const { data: form, isLoading } = useAdminForm()
+
+  return (
+    <>
+      <CategoryHeader mb={0}>Customise Thank You page</CategoryHeader>
+      <Skeleton isLoaded={!isLoading && !!form}>
+        {form && <EndPageSettingsInput endPage={form.endPage} />}
+      </Skeleton>
```

Comment:
skeleton still doesn't show anything, might not even want to wrap if skeleton if not actually using a skeleton

Reply:
Okay, I'll remove the Skeleton for this then!